R/pkg/tests/fulltests/test_utils.R

```diff
@@ -101,7 +104,10 @@ test_that("cleanClosure on R functions", {
   # nolint end
   expect_true("g" %in% ls(env))
   expect_true("l" %in% ls(env))
-  expect_true("f" %in% ls(env))
+  # Enable once SPARK-30629 is fixed
+  # nolint start
+  # expect_true("f" %in% ls(env))
```

Comment:
I remember codes in comments break one of the R linter rule. Might be good to double check.

Reply:
They do, that's why it is wrapped with `nolint`. If I didn't mess up anything, this is cherry picked from previous version of SPARK-23435 proposal, and already passed all tests, but let's see how it goes.